ntoskrnl/ke/amd64/thrdini.c

```diff
@@ -134,7 +134,7 @@ KiInitializeContextThread(IN PKTHREAD Thread,
 
     /* Set up the Context Switch Frame */
     CtxSwitchFrame->Return = (ULONG64)KiThreadStartup;
-    CtxSwitchFrame->ApcBypass = FALSE;
+    CtxSwitchFrame->ApcBypass = TRUE;
```

Comment:
The x64 ApcBypass is not a BOOLEAN, but a KIRQL:
https://git.reactos.org/?p=reactos.git;a=blob;f=sdk/include/ndk/amd64/ketypes.h;hb=5645f7cf8f5b9f41de782535081983db0a7cfb35#l1054
so it should be:
```suggestion
    CtxSwitchFrame->ApcBypass = APC_LEVEL;
```